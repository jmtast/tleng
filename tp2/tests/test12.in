// Error de sem√°ntica: Octava mayor a 9
#tempo negra 120
#compas 2/4

const oct1 = 5;

voz (0)
{
  compas
  {
    nota(re, 10, blanca);
  }
}
