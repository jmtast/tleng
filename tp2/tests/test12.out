Error:
Octava incorrecta. Debe estar entre 1 y 9. LÃ­nea 11
